typescript/test/testdata/translate/package.json.baseline

```diff
@@ -24,14 +24,17 @@
     "mv": "^2.1.1",
     "ncp": "^2.0.0",
     "tmp": "^0.1.0",
-    "typescript": "^3.7.0"
+    "typescript": "^3.7.0",
+    "jsdoc": "^3.5.5",
+    "jsdoc-fresh": "^1.0.1"
   },
   "scripts": {
     "lint": "gts check",
     "clean": "gts clean",
     "compile": "tsc -p . && cp -r protos build/",
     "compile-protos": "compileProtos src",
     "fix": "gts fix",
+    "docs": "jsdoc -c .jsdoc.js",
```

Comment:
Getting closer 😆 @alexander-fenster do we want to include linkinator and it's config, since we're using those everywhere as well?

Reply:
Yes, makes sense. @xiaozhenliu-gg5 can you add `linkinator` to dependencies and scripts (like [here](https://github.com/googleapis/nodejs-pubsub/blob/master/package.json)), and its config like [this one](https://github.com/googleapis/nodejs-pubsub/blob/master/linkinator.config.json)?

Reply:
Sure, added the dependency and tested it work